CodingSession/ImageExtractinRTfield.py

```diff
@@ -112,26 +76,31 @@
             #    html_file.close()
 
             #  replace <img> html tags into explanations text
-            new_text = re.sub(r'<img.+' + re.escape(each_url_one) + r'.+?</img>',
+            new_text = re.sub(r'<img.+' + re.escape(each_url) + r'.+?</img>',
                               '(Please refer to the following case note for screenshot - ' +
                               refid + file_type + ')', new_text)
-    if image_url_two:
+    return new_text, MD5Hash_list
+
+
+def getbase64encodedimage(new_text, md5hash_list):
```

Comment:
There's really no point in passing in md5hash_list. Instead this method should declare its OWN copy of a md5hash_list and return it at the end.

Reply:
I want to create ONE general md5hash_list to loop through and insert/update into database.

Reply:
get_base64_encoded_image can RETURN list and the CALLING method can ADD the results of get_base64_encoded_image to the list that maintained by the calling method.